test/dhcp/record_test.rb

```diff
@@ -59,4 +56,17 @@ def test_should_not_save_invalid_subnets
   def test_options_should_be_a_hash
     assert_kind_of Hash, @record.options
   end
+
+  def test_equality
+    assert_equal Proxy::DHCP::Record.new(:subnet => @subnet, :ip => @ip, :mac => @mac, :option1 => 'one'),
+                 Proxy::DHCP::Record.new(:subnet => @subnet, :ip => @ip, :mac => @mac, :option1 => 'one')
+    assert Proxy::DHCP::Record.new(:subnet => @subnet, :ip => @ip, :mac => @mac, :option1 => 'one') !=
```

Comment:
prefer `assert_not_equal` here and also in the subnet test - like `assert_equal` above, it'll show better error messages when the test fails than true/false.